composer.json

```diff
@@ -32,7 +32,7 @@
         "icecave/parity": "1.0.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.2.20||~2.15.1",
```

Comment:
Allowing the installation of different versions will result in different results when running

```
$ composer style-fix
```

which is probably not what we want.